### ISSUE TITLE
Repository Statistics showing empty charts.

### ISSUE THREAD

**Issue by rogerwcpt** [ForkIssues/Tracker#1725]
I have a few very old repositories and their statistics are fine.  However for repositories created in the last 6 months or so with large amounts of commits, don't have any history.  The repo is my company Git repo, which is from Azure DevOps.

![image](https://user-images.githubusercontent.com/13134927/196352861-0538ae69-3fbb-436d-9ea8-61598d508d60.png)

![image](https://user-images.githubusercontent.com/13134927/196352881-df6ec3ab-1719-4248-be55-423a3f38d1de.png)

**Comment by DominikPalo** [comment]
I'm having the same issue... The issue doesn’t affect all repositories - the statistics functionality works fine in some repositories, but in others, it doesn’t work. In my case, I don't see any correlation with the age of repositories or the number of commits.


The issue is still present in the latest version (2.25.5)

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/1623257/216266547-c0385a79-7455-4931-906b-3eb03ac1c8a7.png">


I can see many JS errors in the `~/Library/Logs/Fork.log` log file:
```
⚠️[RepositoryStatisticsViewController.swift:180] > Error Domain=WKErrorDomain Code=4 "A JavaScript exception occurred" UserInfo={WKJavaScriptExceptionLineNumber=1, WKJavaScriptExceptionMessage=ReferenceError: Can't find variable: updateLineChart, WKJavaScriptExceptionColumnNumber=16, WKJavaScriptExceptionSourceURL=undefined, NSLocalizedDescription=A JavaScript exception occurred}
```

**Comment by DominikPalo** [comment]
@DanPristupov do you have any updates? It used to be a really useful feature, but it still doesn't work on most of our repositories 😟

**Comment by DanPristupov** [comment]
@DominikPalo can you find any open repository which has a problem with statistics, so I could reproduce the problem?

**Comment by DominikPalo** [comment]
Of course, for example, this one: https://github.com/slidoapp/oauth2-swift/

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/1623257/234813845-0d5aa274-05c2-498d-bf2b-6c155c08e83a.png">

**Comment by DanPristupov** [comment]
MacOS 12.6.3 :(
<img width="1063" alt="Screenshot 2023-04-27 at 11 27 23" src="https://user-images.githubusercontent.com/618115/234820548-340323b3-08fd-4808-9ea3-9b3898753eff.png">

I'll try on macOS 13.

**Comment by DanPristupov** [comment]
Could you check if the statistics is fixed in Fork 2.30?

**Comment by DominikPalo** [comment]
@DanPristupov thank you, I've just tested v2.30 and the initial attempt failed as before - with an empty chart, so I closed the statistics window, reopened it again for the same repo and the second attempt worked. So, it appears that the issue is not completely resolved yet

Logs:
```
 /$$$$$$$$                     /$$
| $$_____/                    | $$             Fork 2.30
| $$        /$$$$$$   /$$$$$$ | $$   /$$
| $$$$$    /$$__  $$ /$$__  $$| $$  /$$/       Git: internal
| $$__/   | $$  \ $$| $$  \__/| $$$$$$/        Update Channel: develop
| $$      | $$  | $$| $$      | $$_  $$
| $$      |  $$$$$$/| $$      | $$ \  $$          https://fork.dev
|__/       \______/ |__/      |__/  \__/
.
.
.
⚠️[RepositoryStatisticsViewController.swift:180] > Error Domain=WKErrorDomain Code=4 "A JavaScript exception occurred" UserInfo={WKJavaScriptExceptionLineNumber=1, WKJavaScriptExceptionMessage=ReferenceError: Can't find variable: updateLineChart, WKJavaScriptExceptionColumnNumber=16, WKJavaScriptExceptionSourceURL=undefined, NSLocalizedDescription=A JavaScript exception occurred}
```

**Comment by DominikPalo** [comment]
@DanPristupov it seems the issue is back - I have version 2.34.1 and the chars are empty again 😟


<img width="1023" alt="image" src="https://github.com/fork-dev/Tracker/assets/1623257/029f247a-9efc-4a2b-8886-3f55196194ae">

```
⚠️[RepositoryStatisticsViewController.swift:180] > Error Domain=WKErrorDomain Code=4 "A JavaScript exception occurred" UserInfo={WKJavaScriptExceptionLineNumber=1, WKJavaScriptExceptionMessage=ReferenceError: Can't find variable: updateLineChart, WKJavaScriptExceptionColumnNumber=16, WKJavaScriptExceptionSourceURL=undefined, NSLocalizedDescription=A JavaScript exception occurred}
```

**Comment by DominikPalo** [comment]
Sorry for pinging again - are there any plans/ETA to fix the issue? It's still a problem for us, we are always on the latest version, but the Statistics doesn't work for most of our repos.

BTW, closing the empty "Statistics" window and reopening it helps most of the time - only the first attempt always fails. So, at least we have a workaround. Based on this behavior, I assume it shouldn't be a difficult fix.
